equinix/data_source_metal_device_bgp_neighbors.go

```diff
@@ -126,13 +126,13 @@ func getRoutesSlice(routes []packngo.BGPRoute) []map[string]interface{} {
 	return ret
 }
 
-func getBgpNeighbors(ns []packngo.BGPNeighbor) []map[string]interface{} {
+func getBgpNeighbors(ns *metalv1.BgpSessionNeighbors) []map[string]interface{} {
 	ret := make([]map[string]interface{}, 0, 1)
-	for _, n := range ns {
+	for _, n := range ns.BgpNeighbors {
 		neighbor := map[string]interface{}{
 			"address_family": n.AddressFamily,
 			"customer_as":    n.CustomerAs,
-			"customer_ip":    n.CustomerIP,
+			"customer_ip":    n.CustomerIp,
```

Comment:
This should probably be `n.GetCustomerIp()` (and the other items in this map should all probably be `n.GetX()`), since the metal-go attributes are pointers and the packngo attributes were not.